v2/internal/frontend/desktop/linux/webkit2_36.go

```diff
@@ -0,0 +1,73 @@
+//go:build linux && webkit2_36
+
+package linux
+
+/*
+#cgo linux pkg-config: gtk+-3.0 webkit2gtk-4.0 libsoup-2.4
```

Comment:
Is `libsoup` a new dependency that we need to add to our website and package manager code?

Reply:
WebKit2GTK uses `libsoup` internally, so I think it should not be possible to have WebKit2GTK installed without this dependency. But maybe it's better to be on the safe side and add it.

I'll add that as soon as we have merged the CLI refactoring so it doesn't interfere with that. Are you fine with that?

Reply:
Yeah sure, though I suspect you are right: it's probably installed by webkit2gtk anyway so no need to specify it 👍